themes/default/content/blog/pulumi-release-notes-68/index.md

```diff
@@ -0,0 +1,128 @@
+---
+title: "Feb. 11 Releases: Update Plans public preview, Helm Release for Kubernetes GA and new Pulumi Service sign-in experience"
```

Comment:
Perhaps this is consistent with precedent - but feels a little odd to be explicit about the date but two weeks before the blog post. I can imagine just leaving off the date, saying just February, or citing the milestone instead of the date.